### PR TITLE
Use multicall to fetch dAPI batches

### PR DESCRIPTION
**Pull request by Siegrift**
Closes https://github.com/api3dao/airseeker-v2/issues/46

## Rationale

I first wanted to only fix the issue of multicalling the first batch together with the dAPIs count. I wanted to verify that I implemented the functionality correctly, so I wanted to write an e2e test for this (to test the on-chain integration). I followed the setup implemented by @acenolaza [here](https://github.com/api3dao/dapi-management/pull/3/files#diff-731d7bf404aa3af378fe6396cbdbd4328ee7f69d5723adec8f2ed5fdb99bb44cR60) but the contract we used in Airseeker was a bit outdated so I needed to update it. This required using the multicall everywhere.

### CODE REVIEW

File: jest-e2e.config.js [hunk 1]
```diff
@@ -15,5 +15,6 @@ module.exports = {
   testEnvironment: 'jest-environment-node',
   testMatch: ['**/?(*.)+(feature).[t]s?(x)'],
   testPathIgnorePatterns: ['<rootDir>/.build', '<rootDir>/dist/', '<rootDir>/build/'],
+  testTimeout: 15_000,
```
Comment (Siegrift) [inline jest-e2e.config.js:18]: This will set 15 seconds timeout for all e2e tests, which should be enough.

File: src/env/env.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ let env: EnvConfig | undefined;
 export const loadEnv = () => {
   if (env) return env;
 
-  dotenv.config({ path: join(__dirname, '../.env') });
+  dotenv.config({ path: join(__dirname, '../../.env') });
```
Comment (Siegrift) [inline src/env/env.ts:12]: This previously pointed to a non-existent file.

File: test/setup/contract.ts [hunk 3]
```diff
@@ -0,0 +1,370 @@
+import * as abi from '@api3/airnode-abi';
```
Comment (Siegrift) [inline test/setup/contract.ts:1]: This file is heavily inspired by https://github.com/api3dao/airseeker/blob/main/test/setup/deployment.ts and https://github.com/api3dao/dapi-management/pull/3/files#diff-731d7bf404aa3af378fe6396cbdbd4328ee7f69d5723adec8f2ed5fdb99bb44cR60

**Review by vponline — APPROVED** [review]
LGTM, I didn't see any issues 🚀

**Pull request merged**